pkg/pgmodel/ingestor/ingestor.go

```diff
@@ -86,6 +89,22 @@ func (ingestor *DBIngestor) IngestTraces(ctx context.Context, traces ptrace.Trac
 	return ingestor.tWriter.InsertTraces(ctx, traces)
 }
 
+func (ingestor *DBIngestor) IngestJaegerSpan(ctx context.Context, span *jaegermodel.Span) error {
+	if ingestor.closed.Load() {
+		return fmt.Errorf("ingestor is closed and can't ingest traces")
+	}
+	batches := []*jaegermodel.Batch{
+		{
+			Spans: []*jaegermodel.Span{span},
```

Comment:
Shouldn't we batch here for some seconds or a timeout and then ingest in `tWriter.InsertTraces()`?

Reply:
@Harkishen-Singh , batching is a separate task which @niksajakovljevic handles it for both otel and jaeger spans.